docs/source/conf.py

```diff
@@ -70,7 +70,7 @@
     'IPython.sphinxext.ipython_directive',
     'IPython.sphinxext.ipython_console_highlighting',
     'breathe',
-    'sphinx_tabs.tabs'
+    # 'sphinx_tabs.tabs'
```

Comment:
Just curious, did this make trouble that you needed to comment it out?

Reply:
Whoops, didn't mean to commit that. Yeah, the reason is that I had uninstalled it for updating sphinx (I should reinstall sphinx-tabs with pip, the latest version if not yet available with conda). Will revert this